### ISSUE TITLE
Generate Forever Not Updating Live Preview, Tab Title, or Progress Bar Properly 

### ISSUE THREAD

**Issue by jahaynes256** [lllyasviel/stable-diffusion-webui-forge#875]
After the most recent update, these are not updating properly. Images are being generated but the preview does not show the most recent generated image, progress bar only says waiting, and the tab in firefox rapidly oscillates from show status to say "Stable Diffusion."

**Comment by joshpeterson1** [comment]
I'm having a similar issue where once I use 'generate forever', 'cancel generate forever' does not have an effect, and forge continues generating indefinitely despite showing the 'Generate' button and no progress par. The images continue popping up in the output folder and in the UI as well. 

I've also noticed that since this update, changes I make to the prompt while generating forever are never reflected, almost like the UI has become disconnected from the application itself even though image outputs are shown.

**Comment by devilkad** [comment]
Generate button and live preview all broken, the checkpoints and loras and textual inversion images are broken

**Comment by Blueblizard** [comment]
Yes, i have the same problem since the update. The images are created but the live previews are broken. With the first image it works well but eventually the progress bar and previews stop working and have similar problems

**Comment by Leayal** [comment]
Do you have extension [sd-forge-couple](https://github.com/Haoming02/sd-forge-couple) installed?
If you do, try to remove the extension (not just disable it in settings) and restart Forge and see if the slow down is still persist or not.
If there are other third-party or community-made extensions as well, can you try remove/disable them, too.
From my side, it seems that any extensions that have a high rate of "state sync" requests will likely degrade the performance of Gradio 4.
Check `Network` tab of `DevTools` of your browser (F12 to open devtools on Firefox, I don't know for other browsers).

With WebUI Forge alone, I don't experience UI slow down (which cause extreme delay in live preview updates). Once I start installing `sd-forge-couple`, the issue appears. After observing, I see `sd-forge-couple` perform endless `state sync` requests along with a bunch of PNG image (that is of the extension's preview of regions), this degrade UI performance and cause slow down. The second live preview takes over 5-minute after completing generation to show up on the browser...

**Comment by Blueblizard** [comment]
In my case the extension causing the problem is hypertile integrated. When i activate it, the images are being generated but the preview does not show and the progress bar only says waiting. Also i can´t stop generation.

**Comment by jahaynes256** [comment]
I don't have hypertile, but I forgot to mention I also have the same behavior as Blueblizzard. Images still are being generated, the preview is broken, and I can't interrupt or cancel it. The normal (old) behavior allows me to change prompt before generation without canceling, but now it just goes on for a very long time with the same prompt it bugged out at. It does end eventually if I stop it quickly. 

It's as if it sent requests very quickly before generations finish into a long queue.

I think this behavior is coming from going to other tabs than txt2img, but not sure.

**Comment by Blueblizard** [comment]
> I think this behavior is coming from going to other tabs than txt2img, but not sure.

Yes, abandoning the txt2img tab starts the problem in mi case too.

**Comment by rec32** [comment]
I suffer from the same problem, but it happens even for single generation. I use bare txt2img.

**Comment by TroubleDarkness** [comment]
Hi, the same problem here. Works for the latest Forge WebUI version at least. Happens after switching tabs, as said above. Extensions doesn't matter. Turned all on - off, nothing changed. Switching tabs breaks progress bar, preview and generate forever feature. Is there any possible solutions? Will it be fixed? Or we just need to cope a little bit?

**Comment by Ryoidenshii** [comment]
I have the exact same issue, but to add more, it seems like my UI breaks after some time even if I don't switch tabs.

**Comment by matt-c1** [comment]
I posted steps to reproduce here, it's very simple to break:
https://github.com/lllyasviel/stable-diffusion-webui-forge/issues/1292#issuecomment-2299240282
Another user proposed a partial solution, but not as a pull request because it's not a full fix:
https://github.com/lllyasviel/stable-diffusion-webui-forge/issues/1292#issuecomment-2304870336

**Comment by Blueblizard** [comment]
¿Some solution?
